src/Pages/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -237,7 +237,11 @@ export default function SnapshotListModal() {
                     }: SnapshotItem,
                     index: number,
                   ) => (
-                    <Tr key={created_at + index}>
+                    <Tr
+                      key={created_at + index}
+                      ouiaId={'snapshot_row_' + index}
```

Comment:
no need for the index actually, it'll resolve within the parent element's context for the table row.